lib/engine/game/g_system18/map_britain_customization.rb

```diff
@@ -0,0 +1,497 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module GSystem18
+      module MapBritainCustomization
+        BRITAIN_REGION_HEXES = {
+          'Scotland' => %w[A6 A8 A10 B3 B5 B7 B9 C4 C6],
+          'Wales' => %w[G4 H3 I2 I4 J3],
```

Comment:
For a micro-optimisation, you could shorten both of these arrays by only including the hexes with cities and off-board areas as they're the only ones that count for the cross-border bonuses.

Reply:
Eh, I could, but I actually like the tiny bit of self-documentation this provides. Also, rules and maps change.